packages/nuxeo-web-ui-ftest/pages/ui/bulk_edit.js

```diff
@@ -0,0 +1,69 @@
+import BasePage from '../base';
+
+export default class BulkEdit extends BasePage {
+  get editDocumentsButton() {
+    return this.el;
+  }
+
+  get bulkDefaultLayout() {
+    return this.el.element('#dialog nuxeo-bulk-default-layout');
+  }
+
+  get dialog() {
+    return this.el.element('#dialog');
+  }
+
+  get cancelButton() {
+    return this.el.element('.actions :first-child');
+  }
+
+  get saveButton() {
+    return this.el.element('.actions #save');
+  }
+
+  getField(field) {
+    return this.el.$(`[name="${field}"]`);
+  }
+
+  getFieldValue(field) {
+    const fieldEl = this.getField(field);
+    return fixtures.layouts.getValue(fieldEl);
+  }
+
+  setFieldValue(field, value) {
+    const fieldEl = this.getField(field);
+    fieldEl.waitForVisible();
+    return fixtures.layouts.setValue(fieldEl, value);
+  }
+
+  editMultipleOptions(table) {
+    table.rows().forEach((row) => {
+      const fieldName = row[0];
+      const fieldValue = row[1];
+      const action = row[2];
```

Comment:
nit: can be conciser with object destructuring:
```suggestion
      const [ fieldName, fieldValue, action ] = row;
```